.docker-hub/print/Dockerfile

```diff
@@ -6,7 +6,7 @@ COPY common /common
 WORKDIR /app
 
 COPY print/package*.json ./
-RUN mkdir -p /home/node/.cache/node-gyp/18.7.0 && npm ci
```

Comment:
This directory had to be added, along with some other adjustments, in order to support node 18: 412951e70e8944eb57bd4111af2000b2bed186e9
Is this really not needed anymore? And if so, can some of the other changes from that commit also be removed?

Reply:
For me it ran without this folder locally and on CI. We're also using node 18.14.2 in the meantime and not 18.7.0 anymore, so I think if this would really be needed, we should have noticed in the meantime.

Also reverted some of the other changes from commit https://github.com/ecamp/ecamp3/commit/412951e70e8944eb57bd4111af2000b2bed186e9

Reply:
Verified it by deploying to feature branch deployment.